### DISCUSSION TITLE
Firefox for Android revisited

### DISCUSSION THREAD

**Discussion by codethief** [jordwest/news-feed-eradicator#381] [Ideas]
I'd like to pick up an old issue again, namely #21 (and #35). Firefox for Android has since seen support for Web Extensions and I'd like to know whether it'd be possible now to eradicate the news feed from the mobile version of Facebook? :) (I installed NFE on my Android phone but m.facebook.com is still showing the news feed.)

**Comment by mohammedi-haroune** [comment]
It would be great to have that, I'll happy to work on it @jordwest

**Comment by jordwest** [comment]
Hi @mohammedi-haroune, that'd be really great! I've started reorganising the project to prepare  for the new facebook beta (see #82) so there's now different scripts for removing different sites, see: https://github.com/jordwest/news-feed-eradicator/tree/master/src/sites. Just make a copy of one of those and call it something like `fb-mobile.ts`.

You'll also need to update this logic to add the new case, probably checking the domain matches `m.facebook.com` is enough for now:
https://github.com/jordwest/news-feed-eradicator/blob/cd3cb60912c963d540aba80d8d8db5d42267fbcc/src/eradicate.ts#L9

**Comment by mochsner** [comment]
Note that Firefox just last year moved to using a new engine ("GeckoView") on Android. 


See here: https://extensionworkshop.com/documentation/develop/developing-extensions-for-firefox-for-android/

With that in mind, has there been any progress on this issue? I'm new to plug-in development, but I'd love to see this. If there's any good resources I can check-out for how to get up-to-date with development of this plugin on other browsers (particularly chrome on Android, if that's available), I might be able to take a stab at it.

**Comment by codethief** [comment]
@jordwest I see you converted the original issue #67 into a discussion. Meanwhile, there has been [an open PR](https://github.com/jordwest/news-feed-eradicator/pull/293) to address said issue for a while. Is there anything preventing it from getting reviewed & merged?

**Reply by jordwest** [reply]
Merged!
